1-exercises/F-array-forEach/exercise.js

```diff
@@ -7,8 +7,17 @@
   An array with numbers 1-15 has been provided.
 */
 
-let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
-
+let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];{
+const newArr = arr.forEach(item => {
```

Comment:
You can think about forEach method like about for loop. Please find more about it here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
https://www.w3schools.com/jsref/jsref_foreach.asp

please check this as well:
https://stackoverflow.com/questions/2363840/how-to-use-or-condition-in-a-javascript-if-statement